pulsar-client-go/pulsar/producer.go

```diff
@@ -163,6 +163,14 @@ type Producer interface {
 	// the eventual error in publishing
 	SendAsync(context.Context, ProducerMessage, func(ProducerMessage, error))
 
+	// Get the last sequence id that was published by this producer.
+	// This represent either the automatically assigned or custom sequence id (set on the MessageBuilder) that
```

Comment:
Instead of `MessageBuilder` which is in Java client lib, we should refer to `ProducerMessage` here (https://github.com/apache/pulsar/blob/master/pulsar-client-go/pulsar/message.go#L24)

Reply:
Also, the `ProducerMessage` doesn't currently have a way to specify the sequence id of a particular message. This should be done by using `pulsar_message_set_sequence_id()` function.

Reply:
yes, you are reight. I will provide this interface to achieve the corresponding function in next pull request.